lib/main.dart

```diff
@@ -1,67 +1,315 @@
+        decoration: BoxDecoration(
+            border: Border.all(
+              color: Colors.indigo,
+            ),
+            borderRadius: const BorderRadius.all(Radius.circular(100)),
+            image: DecorationImage(image: AssetImage(link), fit: BoxFit.cover)),
+      ),
+      Row(children: [
+        Align(
+            alignment: Alignment.bottomCenter,
+            child: Text(
+              gameName,
+              style: const TextStyle(fontSize: 18, color: Color(0xFFf2f2f2)),
+            )),
+        const Icon(
+          Icons.verified,
+          size: 18,
+          color: Color(0xFFf2f2f2),
+        )
+      ]),
+    ]),
+  );
+}
+
+Widget _imageContainerThird(BuildContext context, String link) {
+  var phoneSize = MediaQuery.of(context).size;
+  if (MediaQuery.of(context).orientation == Orientation.landscape) {
+    orientationWidth = 0.7;
+    orientationHeight = 2.1;
+    marginDivider = 2;
+  }{
+    orientationWidth = 1;
+    orientationHeight = 1;
+    marginDivider = 1;
+  }
+  return Container(
+    width: phoneSize.width * 0.42 * orientationWidth,
+    height: phoneSize.height * 0.23 * orientationHeight,
+    margin: const EdgeInsets.all(16.0),
+    decoration: BoxDecoration(
+        borderRadius: const BorderRadius.all(Radius.circular(5.5)),
+        image: DecorationImage(image: AssetImage(link), fit: BoxFit.cover)),
+  );
+}
+
+Widget _textContainer(
+    BuildContext context, String mainText, String secondText) {
+  return Container(
+    margin: const EdgeInsets.all(12.0),
+    child: Row(
+      children: [
+        Text(
+          mainText,
+          style: const TextStyle(fontSize: 23, color: Color(0xFFf2f2f2)),
+        ),
+        Text(
+          secondText,
+          style: const TextStyle(fontSize: 12, color: Color(0xFFf2f2f2)),
+        )
+      ],
+    ),
+  );
+}
+
+Widget _iconBottom(BuildContext context, IconData icon, double height) {
+  return Icon(
+    icon,
+    size: height * 0.055,
+    color: Color(0xFFf2f2f2),
+  );
+}
+
+Widget _iconAppBar(BuildContext context, String icon, double width) {
+  if (MediaQuery.of(context).orientation == Orientation.landscape) {
+    orientationWidth = 0.4;
+
+  }
+  return Container(
+    margin: const EdgeInsets.only(
+      left: 10.0,
+      right: 10.0,
+    ),
+    height: width * 0.125*orientationWidth,
+    width: width * 0.125*orientationWidth,
+    decoration: BoxDecoration(
+        borderRadius: const BorderRadius.all(Radius.circular(100)),
+        image: DecorationImage(image: AssetImage(icon), fit: BoxFit.cover)),
+  );
+}
+
+class _HomePageState extends State<HomePage> {
+  Widget _horizontalListMain(BuildContext context) {
+    return Row(children: [
+      Expanded(
+        flex: 1,
+        child: SingleChildScrollView(
+          scrollDirection: Axis.horizontal,
+          child:
+              Row(mainAxisAlignment: MainAxisAlignment.end, children: <Widget>[
+            _imageContainerStart(context, 'assets/img_1.png',
```

Comment:
Don't forget that you're still using OOP (even if it's Dart).
Do Something like
```
var games = [
  { title: "Need for Speed Payback", description: "Jump back in", icon: "assets/img_1.png" },
  ...
];
```

and then build your layout dynamically by iterating through this array